src/SignInComponent.tsx

```diff
@@ -0,0 +1,342 @@
+import React, { useState } from 'react';
+import { css } from "@emotion/core";
+import { body, headline } from '@guardian/src-foundations/typography/cjs';
+import { palette } from "@guardian/src-foundations";
+import { Button } from "@guardian/src-button";
+import { from, until } from '@guardian/src-foundations/mq';
+import { space } from '@guardian/src-foundations';
+import { SvgCross } from '@guardian/src-icons';
+import { SvgInfo } from '@guardian/src-icons';
```

Comment:
These could be combined into a single import:

```js
import { SvgCross, SvgInfo } from '@guardian/src-icons';
```

I think it's clearer that way, what do you reckon?

(This is also the case with the `src-foundation` import.)

Reply:
Good shout, will do!